wireguard/wireguard_windows.go

```diff
@@ -83,18 +83,18 @@ func SetRoutes(addrs []ifaceAddress) {
 		if addr.Network.IP.To4() != nil {
 			slog.Info("adding ipv4 route to interface", "route", fmt.Sprintf("%s -> %s", addr.IP.String(), addr.Network.String()))
 			cmd := fmt.Sprintf("netsh int ipv4 add route %s interface=%s nexthop=%s store=%s",
-				addr.Network.String(), ncutils.GetInterfaceName(), IPv4Network, "active")
+				addr.Network.String(), ncutils.GetInterfaceName(), "0.0.0.0", "active")
 			_, err := ncutils.RunCmd(cmd, false)
-			if err != nil {
-				slog.Error("failed to apply", "ipv4 egress range", addr.Network.String())
+			if err != nil && !strings.Contains(err.Error(), "exit status 1") {
```

Comment:
if command exits with exit status 1, that's an error. why are we ignoring it here?